### DISCUSSION TITLE
Failed to pull docker

### DISCUSSION THREAD

**Discussion by weiwuwei** [tangyoha/telegram_media_downloader#76] [Q&A] [answered]
When I run `% docker pull tangyoha/telegram_media_downloader:latest`

It shows
`latest: Pulling from tangyoha/telegram_media_downloader
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'`

Would you please help to fix? Thanks!

**Comment by tangyoha** [comment]
you can use `docker-compose up --build`

**Reply by weiwuwei** [reply]
```
% docker-compose up --build
Building telegram_media_downloader
ERROR: Cannot locate specified Dockerfile: Dockerfile
```

Do I need to modify anything in docker-compose.yaml? Sorry, I am new to docker

**Reply by tangyoha** [reply]
no ，just use this command to compile docker image

**Reply by weiwuwei** [reply, marked as answer]
Thanks! The issue has been resolved after I upgrade my docker to latest version and use `colima`
